src/components/AlbumsList.jsx

```diff
@@ -0,0 +1,21 @@
+import styles from '../styles/albumsList'
+
+import React from 'react'
+import { translate } from '../lib/I18n'
```

Comment:
As far as I can tell, the translation module isn't used here. Maybe we can remove it?